tests/util.go

```diff
@@ -60,3 +60,7 @@ func waitForPodNotFoundInNamespace(c kubernetes.Interface, podName, ns string, t
 		return false, nil
 	})
 }
+
+func Int32Ptr(i int32) *int32 {
```

Comment:
Where it is used? Why not use `k8s.io/utils/pointer` package

Reply:
https://github.com/lizhemingi/tidb-operator/blob/215681ba23c368097acb8a76b744a3eff3ec571f/tests/e2e/tikv/tikv.go#L358
It's used here.

Oh yeah I think it can be replaced by `k8s.io/utils/pointer`. Will do it later.